frontend/util/animations/aperture.json

```diff
@@ -0,0 +1 @@
shapes":[{"ty":"gr","it":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":0,"k":{"i":[[16.718,1.007],[58.209,-7.05],[58.209,-19.235],[52.066,-33.334],[55.792,-44.513],[55.792,-52.872],[44.614,-71.503],[7.654,-63.748],[7.654,-52.67],[13.193,-39.477],[5.237,-27.494],[5.237,-18.933]],"o":[[46.728,1.007],[58.209,-19.034],[58.209,-27.594],[50.253,-39.477],[55.792,-52.67],[55.792,-63.748],[18.832,-71.503],[7.654,-52.872],[7.654,-44.513],[11.28,-33.435],[5.237,-19.135],[5.237,-7.15]],"v":[[31.723,1.007],[58.209,-19.034],[58.209,-19.235],[43.707,-36.456],[55.792,-52.67],[55.792,-52.872],[31.723,-71.503],[7.654,-52.872],[7.654,-52.67],[19.739,-36.456],[5.237,-19.135],[5.237,-18.933]],"c":true}},"nm":"8","mn":"ADBE Vector Shape - Group"},{"ind":1,"ty":"sh","ix":2,"ks":{"a":0,"k":{"i":[[40.686,-39.276],[15.61,-44.513],[15.61,-52.368],[22.559,-64.554],[47.836,-59.317],[47.836,-52.167]],"o":[[22.76,-39.276],[15.61,-52.167],[15.61,-59.418],[40.887,-64.554],[47.836,-52.368],[47.836,-44.513]],"v":[[31.723,-39.276],[15.61,-52.167],[15.61,-52.368],[31.723,-64.554],[47.836,-52.368],[47.836,-52.167]],"c":true}},"nm":"8","mn":"ADBE Vector Shape - Group"},{"ind":2,"ty":"sh","ix":3,"ks":{"a":0,"k":{"i":[[43.304,-5.942],[13.293,-12.085],[13.293,-19.336],[21.451,-32.73],[50.153,-27.191],[50.153,-19.135]],"o":[[20.142,-5.942],[13.293,-19.135],[13.293,-27.191],[41.995,-32.73],[50.153,-19.336],[50.153,-11.985]],"v":[[31.723,-5.942],[13.293,-19.135],[13.293,-19.336],[31.723,-32.73],[50.153,-19.336],[50.153,-19.135]],"c":true}},"nm":"8","mn":"ADBE Vector Shape - Group"}],"nm":"8","np":6,"cix":2,"ix":1,"mn":"ADBE Vector Group"},{"ty":"gr","it":[{"ind":0,"ty":"sh","ix":1,"ks":{"a":0,"k":{"i":[[-15.005,0],[0,11.984],[0,0],[8.359,3.122],[0,8.157],[0,0],[12.891,0],[0,-10.876],[0,0],[-6.546,-3.021],[0,-8.359],[0,0]],"o":[[15.005,0],[0,0],[0,-8.359],[6.546,-3.021],[0,0],[0,-10.876],[-12.891,0],[0,0],[0,8.157],[-8.459,3.021],[0,0],[0,11.783]],"v":[[94.722,1.007],[121.208,-19.034],[121.208,-19.235],[106.706,-36.456],[118.791,-52.67],[118.791,-52.872],[94.722,-71.503],[70.653,-52.872],[70.653,-52.67],[82.738,-36.456],[68.236,-19.135],[68.236,-18.933]],"c":true}},"nm":"8","mn":"ADBE Vector Shape - Group"},{"ind":1,"ty":"sh","ix":2,"ks":{"a":0,"k":{"i":[[8.963,0],[0,7.654],[0,0],[-9.164,0],[0,-6.949],[0,0]],"o":[[-8.963,0],[0,0],[0,-7.05],[9.164,0],[0,0],[0,7.654]],"v":[[94.722,-39.276],[78.609,-52.167],[78.609,-52.368],[94.722,-64.554],[110.835,-52.368],[110.835,-52.167]],"c":true}},"nm":"8","mn":"ADBE Vector Shape - Group"},{"ind":2,"ty":"sh","ix":3,"ks":{"a":0,"k":{"i":[[11.581,0],[0,7.05],[0,0],[-10.272,0],[0,-7.855],[0,0]],"o":[[-11.581,0],[0,0],[0,-7.855],[10.272,0],[0,0],[0,7.15]],"v":[[94.722,-5.942],[76.292,-19.135],[76.292,-19.336],[94.722,-32.73],[113.152,-19.336],[113.152,-19.135]],"c":true}},"nm":"8","mn":"ADBE Vector Shape - Group"},{"ty":"mm","mm":1,"nm":"Merge Paths 1","mn":"ADBE Vector Filter - Merge"}],"nm":"8","np":6,"cix":2,"ix":2,"mn":"ADBE Vector Group"}]},"fFamily":"Gotham"}]}
```

Comment:
Where is this coming from? If you have created it yourself, adding the source file aswell would be nice, otherwise, what license is it under?

Reply:
These are free Lottie-animations and comes under creative commons license. This is the [link ](https://lottiefiles.com/624-camera-aperture)to the animation, you can see the license at the end of the page. I have changed the colors to match the theme :)